flexera/aws/idle_compute_instances.pt

```diff
@@ -82,6 +172,7 @@ policy "pol_utilization" do
 {{with index data 0}}{{.message}}{{end}}
 EOS
     escalate $email
```

Comment:
cannot see an escalation definition for $email referenced here - not sure whether this is immaterial as this is a mock